tests/e2e/e2e_coredaos_test.go

```diff
@@ -0,0 +1,173 @@
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+
+	coredaostypes "github.com/atomone-hub/atomone/x/coredaos/types"
+	govtypesv1 "github.com/atomone-hub/atomone/x/gov/types/v1"
+	govtypesv1beta1 "github.com/atomone-hub/atomone/x/gov/types/v1beta1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func (s *IntegrationTestSuite) testCoreDAOs() {
+	valIdx := 0
+	s.Run("coredaos parameter change", func() {
```

Comment:
This subtest must be executed for the following ones to succeed. In other words, it's not possible to execute for example `TestIntegrationTestSuite/TestCoreDAOs/coredaos_annotation` if `coredaos parameter change` was not executed before. I think all tests could be executed separately from the others.

A quick fix can be to unwrap `coredaos parameter change` and make it like a kind of setup for the following subtests (by unwrap I mean remove the surrounding `s.Run()` and execute it directly in `testCoreDAOs`).